Client/Frontend/Home/FxHomeJumpBackInViewModel.swift

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+class FirefoxHomeJumpBackInViewModel {
+
+    // MARK: - Properties
+    var jumpableTabs = [Tab]()
+    private var recentTabs = [Tab]()
+
+    var layoutVariables: JumpBackInLayoutVariables
+    var tabManager: TabManager?
+
+    init() {
+        self.tabManager = BrowserViewController.foregroundBVC().tabManager
+        self.layoutVariables = JumpBackInLayoutVariables(columns: 1, scrollDirection: .vertical, maxItemsToDisplay: 2)
+    }
+
+    public func updateDataAnd(_ layoutVariables: JumpBackInLayoutVariables) {
+        self.layoutVariables = layoutVariables
+        loadItems()
+    }
+
+    public func switchTo(_ tab: Tab) {
+        TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .firefoxHomepage, value: .jumpBackInSectionTabOpened)
```

Comment:
nit: Maybe we can move the telemetry call to the end of the function so that, if for whatever reason it doesn't switch tabs, the call isn't made prematurely.